types/src/validator_change.rs

```diff
@@ -1,15 +1,125 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{contract_event::EventWithProof, ledger_info::LedgerInfoWithSignatures};
+use crate::crypto_proxies::ValidatorVerifier;
+use crate::ledger_info::LedgerInfoWithSignatures;
 use failure::*;
+use libra_crypto::ed25519::*;
 use libra_crypto::*;
 use std::convert::{TryFrom, TryInto};
 
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct ValidatorChangeEventWithProof<Sig> {
-    ledger_info_with_sigs: LedgerInfoWithSignatures<Sig>,
-    event_with_proof: EventWithProof,
+    ledger_info_with_sigs: Vec<LedgerInfoWithSignatures<Sig>>,
```

Comment:
Changing this without renaming the struct is very confusing. Do you think we should get rid of the "Event" part in the name?

Reply:
well, there might be a plan to add the event back later so I didn't change the name, but sure we can rename it for now